scripts/core/exp-bindless-images.yml

```
@@ -648,7 +648,7 @@ params:
       desc: "[in] requested level of the mipmap"
     - type: "$x_exp_image_mem_native_handle_t*"
       name: phImageMem
-      desc: "[out] returning memory handle to the individual image"
+      desc: "[out][alloc] returning memory handle to the individual image"
```

```suggestion
      desc: "[out] returning memory handle to the individual image"
```

`MipmapGetLevelExp` doesn't allocate any handles that need to be freed, it returns a handle to a sub-resource, which itself is freed when the top-level resource is freed (it cannot be freed individually).

---

Thanks for checking; updated.